### ISSUE TITLE
Issue initializing ajax loaded inputs

### ISSUE THREAD

**Issue by Braunson** [dericcain/google-address-autocomplete#11]
I'm looking to initialize a second input that's ajax loaded on a page.

First the page loads with one input which get's initiated no problem.
Once the page is finished loading, it will make an Ajax call to include another input that should be initiated and says it is but it doesn't work as expected, nothing happens.

Ideas?

**Comment by dericcain** [comment]
Is there a way that when you dynamically add the inputs that you can, at that time, bind the Address Autocomplete?

**Comment by Braunson** [comment]
@dericgw Yep that's what I'm doing.

**Comment by ghost** [comment]
Do you have the code in Github somewhere where I can take a look at it?

**Comment by Braunson** [comment]
@dericcain Yep https://gist.github.com/Braunson/b1286c2d56e4e0509ea67d19b269cfb3 I'm sure there's room for improvement. The goal was to be able to initialize the autocomplete on any form element with say `data-google-autocomplete` attribute. 

In my use-case I'm using it in a form, there is always a hidden field that  get's filled with the default callback's json encoded response so when submitted, we parse that json data and save it without having to show the user anything but a nice formatted address in the field.

I'm open to any suggestions or improvements but it seems when trying to run the plugin on a dynamically inserted input field, it doesn't work. Thoughts?
